operations/resourcecategories.md

```diff
@@ -0,0 +1,129 @@
+# Resource categories
+
+## Get all resource categories
+
+Returns all categories of resources. This operation uses [Pagination](../guidelines/pagination.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/resourceCategories/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ResourceCategoryIds": [
+        "5c0804f9-d03a-4b13-a57d-b00300781a41",
+        "47d6b462-35ec-467e-a565-b00300781a41"
+    ],
+    "ServiceIds": [
+        "9b3a6c54-63aa-4383-b50e-b0030078184b"
+    ],
+    "UpdatedUtc": {
+        "StartUtc": "2023-05-05T00:00:00Z",
+        "EndUtc": "2023-05-10T00:00:00Z"
+    },
+    "Limitation":{
+        "Count": 10
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). |
```

Comment:
```suggestion
| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns the resource categories for all enterprises within scope of the Access Token. |
```

Reply:
@MikeAdamsMews I will update this text for order items and payments as well in this PR, to be consistent.